src/nvim/ex_cmds.c

```diff
@@ -5399,9 +5404,12 @@ helptags_one (
    * Find all *.txt files.
    */
   dirlen = (int)STRLEN(dir);
-  STRCPY(NameBuff, dir);
-  STRCAT(NameBuff, "/**/*");
-  STRCAT(NameBuff, ext);
+  if (STRLCPY(NameBuff, dir, MAXPATHL) >= MAXPATHL ||
+      STRLCAT(NameBuff, "/**/*", MAXPATHL) >= MAXPATHL ||
+      STRLCAT(NameBuff, ext, MAXPATHL) >= MAXPATHL) {
```

Comment:
binary operators continue on the next line, see http://neovim.io/develop/style-guide.xml

    if (STRLCPY(NameBuff, dir, MAXPATHL) >= MAXPATHL
      || STRLCAT(NameBuff, "/**/*", MAXPATHL) >= MAXPATHL
      || STRLCAT(NameBuff, ext, MAXPATHL) >= MAXPATHL) {